src/components/progressiveImage/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {View, StyleSheet, Animated} from 'react-native';
+import {AnimatedImage} from 'react-native-ui-lib'; //eslint-disable-line
+
+/**
+ * @description: Image component that loads first a small thumbnail of the images, 
+ *               and fades-in the full-sized image with animation once it's loaded
+ * @extends: Animated.Image 
+ * @example: https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ProgressiveImageScreen.js
+ */
+class ProgressiveImage extends React.Component {
+  thumbnailAnimated = new Animated.Value(0);
+  imageAnimated = new Animated.Value(0);
+
+  getThumbnail = () => {
+    const {thumbnailSource, ...props} = this.props;
```

Comment:
Declare `thumbnailSource` prop in the component `static propTypes`
The propTypes is our API. 
We use it also for our docs site and also this is a clear way to see what that component supports.